notebooks/ldb_Feature Engineering_kaggleEX.ipynb

```diff
@@ -0,0 +1,2643 @@
+{
```

Comment:
<p>This cell returns a error. Please delete or fix!</p><p> </p><p>This notebook give us a lot of insights! You improve your skills a lot since the project beginning. But, you have to pay more attention in the documentation and pattern (comments in Portuguese, filenames, csv attributes...)</p><p> </p><p>I suggest you to resolve fast the comments and create some Markdowns cells to improve the documentation (in english plz)!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Already changed the comments </p>